mscore/inspector/inspectorBase.h

```diff
@@ -40,6 +40,7 @@ struct InspectorItem {
       QWidget* w;
       // QToolButton* r;   // reset to default button (if any)
       QWidget* r;   // reset to default button (if any)
+      bool allowSetAsStyle = true;   // show S button in inspector
```

Comment:
Travis doesn't support this?

Reply:
I guess `operator=` is only defined for a `const std::vector<InspectorItem>`

Reply:
Or intializing a struct's members inside a header files is not supported

Reply:
Just strange that AppVeyor seems to allow it

Reply:
Visual Studio allows it too

Reply:
And "musescore deleted a comment from Howard-C 4 minutes ago"?? I was asking how can I solve this without re-defining all `InspectorItem` in the solution.

Reply:
AppVeyor uses Visual Studio ;-)

Reply:
I deleted a comment of mine, maybe this took your reply along with it?

Reply:
The question is still unanswered :)

Reply:
I don't know. MinGW doesn't like this either BTW

Reply:
This should be a C++14 vs. C++11 issue. For MSVC we use the latter, for XCode/Mac the former, for gcc (Linux and MinGW) the gnu variant of the former

Reply:
I am starting to think https://github.com/musescore/MuseScore/pull/5479#pullrequestreview-318309944 is a better solution xD

Reply:
@MarcSabatella What is your opinion?

Reply:
I would assume there is a way to make this work by using a constructor rather than a static initialization.  Doesn't really matter to me, though.

Reply:
I looked into this and an article suggests it cannot be done without overhauling something. List initialization (using braces) can only happen if no constructor is defined within the struct, and if the struct has constructors, then it must be initialized using them. So these two cannot be in the same world, but in this case only a constructor can do the job as list initialization has compiling errors.